Meadowlark-js/backends/meadowlark-postgresql-backend/src/repository/Delete.ts

```diff
@@ -28,19 +28,21 @@ export async function deleteDocumentByDocumentUuid(
 
   try {
     await client.query('BEGIN');
-
+    // Find the alias ids for the document to be deleted
+    const documentAliasIdsResult: QueryResult = await client.query(findAliasIdsForDocumentByDocumentUuidSql(documentUuid));
+    meadowlarkId =
+      documentAliasIdsResult?.rowCount != null && documentAliasIdsResult.rowCount > 0
+        ? documentAliasIdsResult.rows[0].document_id
+        : ('' as MeadowlarkId);
     if (validateNoReferencesToDocument) {
-      // Find the alias ids for the document to be deleted
-      const documentAliasIdsResult: QueryResult = await client.query(findAliasIdsForDocumentByDocumentUuidSql(documentUuid));
-
       // All documents have alias ids. If no alias ids were found, the document doesn't exist
       if (documentAliasIdsResult.rowCount == null || documentAliasIdsResult.rowCount === 0) {
```

Comment:
This check is now on line 34 as well. Would it make sense to combine in some way? Might be just fine the way it is though.